data-es6.js

```diff
@@ -1524,6 +1532,8 @@ exports.tests = [
         firefox51: true,
         chrome40: flag,
         chrome41: true,
+        node012: flag,
```

Comment:
This one was missed?

![screen shot 2016-10-06 at 5 58 13 pm](https://cloud.githubusercontent.com/assets/588473/19172187/84577060-8bee-11e6-8abe-53196508670b.png)

Looks fine with `nvm use 4 && node .`